js/gallery.js

```diff
@@ -12,9 +12,43 @@ window.onload = function chargeImages() {
     "./../assets/images/gallery/8.jpg"
   ];
   let div1 = document.getElementById("divImagesContainer").children;
-  for (let i = 0; i < div1.length; i++) {
-    div1[i].firstElementChild.style.backgroundImage = `url('${
-      imagesGallery[i]
-    }')`;
+  let imagesContainer = document.querySelector('#divImagesContainer');
+
+  for (let i = 0; i < imagesGallery.length; i++) {
+    imagesContainer.innerHTML += `
+  <div class="gallery-item">
+    <div class="gallery-item__images" style="background-image: url(${imagesGallery[i]})"></div>
```

Comment:
this in the end is not seen as an online style, which I suppose is wrong

Reply:
is not a style is a ubication from a part of the body.

Reply:
Overall, we should not use inline styles, but In this case the gallery image it’s treated as a background image, rather than <img />, this is a case where <div style=”background-image: url(....)” /> is acceptable since the elements are build dynamically.